auctions/english-auction/Cargo.toml

```
@@ -0,0 +1,16 @@
+[project]
+name = "english-auction"
+version = "0.0.0"
+authors = ["Fuel Labs <contact@fuel.sh>"]
+edition = "2021"
+license = "Apache-2.0"
+
+[dependencies]
+fuels = { version = "0.25", features = ["fuel-core-lib"] }
+rand = {version = "0.8" }
```

```suggestion
rand = { version = "0.8" }
```

Comments on rand somewhere below